scenarioo-client/app/pipes/metadataTreeListCreator.pipe.ts

```diff
@@ -0,0 +1,33 @@
+import {Injectable, Pipe, PipeTransform} from '@angular/core';
+import {MetadataTreeCreatorPipe} from './metadataTreeCreator.pipe';
+
+@Pipe({
+    name: 'scMetadataTreeListCreator',
+})
+
+@Injectable()
+export class MetadataTreeListCreatorPipe implements PipeTransform {
+
+    constructor(private metadataTreeCreatorPipe: MetadataTreeCreatorPipe) {
+    }
+
+    transform(metadata: any): any {
+
+        const metadataTrees = [];
+
+        Object.keys(metadata).forEach((key) => {
+
+            const value = metadata[key];
+
+            const transformedValue = this.metadataTreeCreatorPipe.transform(value);
+
+            const entry = {
+                key,
```

Comment:
To make it clear I would assign the value of key to field:
`key: key,`

Reply:
This is generating a Circle Ci Error in JS Tests: Expected property shorthand in object literal ('{key}')

Reply:
Good point. I didn't understand the consequences of the code enough. But with the simplification of metadata this code has changed anyway.